purchase_split_wizard/README.rst

```diff
@@ -0,0 +1,97 @@
+=====================
+Purchase Split Wizard
+=====================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:8dca0033c5b25b52fae116e7724643190fea0f92542edfa07f49f4b2f5f25f0a
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-sygel--technology%2Fsy--purchase--workflow-lightgray.png?logo=github
+    :target: https://github.com/sygel-technology/sy-purchase-workflow/tree/16.0/purchase_split_wizard
+    :alt: sygel-technology/sy-purchase-workflow
+
+|badge1| |badge2| |badge3|
+
+This module adds a wizard that allows you to split a purchase into
+multiple ones.
+
+The split method has one partcularity, when you split a purchasethe
```

Comment:
I don't get this bit.

Reply:
I have updated the readme and explained it a bit more